read_input_parser_cvc.l

```diff
@@ -548,6 +550,10 @@ static inline double fltlist_next_token(int * const list_end){
   cvc::g_gauge_step = atoi(yytext);
   if(cvc::myverbose!=0) printf("# [read_input_parser] gauge step size set to %s\n", yytext);
 }
+<SPINPROJECTIONPREFIX>{FILENAME} {
+  strcpy(cvc::spinprojectionfname_prefix,yytext);
```

Comment:
please use strncpy

Reply:
or even better, use snprintf and check if the filename fit into the 200 characters